gst/nnstreamer/nnstreamer_conf.c

```diff
@@ -317,26 +316,23 @@ nnsconf_loadconf (gboolean force_reload)
     memset (&conf, 0, sizeof (confdata));
   }
 
-  /* Read from the conf file first */
-  if (g_path_is_absolute (NNSTREAMER_CONF_FILE)) {
+  /**
+   * Priority of reading a conf file
+   * 1) read from NNSTREAMER_CONF_FILE
+   * 2) read from NNSTREAMER_DEFAULT_CONF_FILE
+   * 3) read from env-var
+   */
+  if (g_path_is_absolute (NNSTREAMER_CONF_FILE) &&
```

Comment:
plz leave the option to check absolute path, because default value sysconfdirson is etc (not /etc).
if developer just configures `$ meson build`, this always refers NNSTREAMER_DEFAULT_CONF_FILE

Reply:
Got it